config/routes.rb

```diff
@@ -0,0 +1,8 @@
+Rails.application.routes.draw do
+  root to: 'blogs#index'
+  resources :blogs
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
```

Comment:
Please add users and posts routes  for `index` and `show` actions to `config/routes.rb`.

Reply:
Users and posts routes for index and show